cmd/manager/main.go

```diff
@@ -110,7 +112,7 @@ func main() {
 
 	// Set default manager options
 	options := manager.Options{
-		Namespace:          namespace,
+		Namespace:          "",
 		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
 	}
 
```

Comment:
Can we get rid of the multi namespace support below?

Reply:
The GitOps operator only supports the `all-namespace` installation mode and it creates resources across multiple namespaces. If the CR is namespace scoped, the controller would only create resources in the namespace of the CR

Reply:
I mean can this block of code safely removed?
```
	// Add support for MultiNamespace set in WATCH_NAMESPACE (e.g ns1,ns2)
	// Note that this is not intended to be used for excluding namespaces, this is better done via a Predicate
	// Also note that you may face performance issues when using this with a high number of namespaces.
	// More Info: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/cache#MultiNamespacedCacheBuilder
	if strings.Contains(namespace, ",") {
		options.Namespace = ""
		options.NewCache = cache.MultiNamespacedCacheBuilder(strings.Split(namespace, ","))
	}
```

Reply:
Removed the unwanted code